src/bootstrap/builder.rs

```diff
@@ -949,10 +949,30 @@ impl<'a> Builder<'a> {
             }
         }
 
-        if let Some(host_linker) = self.linker(compiler.host) {
+        // FIXME: Don't use LLD if we're compiling libstd, since it fails to link it.
```

Comment:
Can you elaborate on this? Seems like a bug?

Reply:
I get an unresolved symbol error. It's a bug in either LLD or rustc/link.exe, not sure which.

Reply:
https://github.com/rust-lang/cargo/issues/7840#issuecomment-579153306

Reply:
Done.